src/CommunityToolkit.Maui/Alerts/Toast/Toast.macios.cs

```diff
@@ -0,0 +1,57 @@
+﻿using CommunityToolkit.Maui.Core.Views;
+using CoreGraphics;
+using UIKit;
+
+namespace CommunityToolkit.Maui.Alerts;
+
+public partial class Toast
+{
+	static ToastView? nativeToast;
+
```

Comment:
Since `nativeToast` is `static`, let's add and implement a `SemaphoreSlim(1,1)` like we did for `Snackbar.macios.cs` to ensure we don't induce any race conditions.

```suggestion
	static readonly SemaphoreSlim semaphoreSlim = new(1, 1);

```